### PR TITLE
Studio docs: review and adjust the content/ structure 

### PR DESCRIPTION
**Pull request by tapadipti**
Partially addresses https://github.com/iterative/studio/issues/2244 (except the screenshots and some other points). I'll work on the screenshots in a separate PR.

### CODE REVIEW

File: content/docs/studio/user-guide/projects-and-experiments/live-metrics-and-plots.md [hunk 1]
```diff
@@ -98,7 +86,7 @@ which are nested under the parent Git commit.
 
 <admon>
 
-The live experiments row for an experiment is displayed only if its parent Git
+The live experiment row for an experiment is displayed only if its parent Git
```
Comment (tapadipti) [inline content/docs/studio/user-guide/projects-and-experiments/live-metrics-and-plots.md:89]: @mvshmakov We can remove this statement now, because of detached experiments, right?

Comment (mvshmakov) [reply content/docs/studio/user-guide/projects-and-experiments/live-metrics-and-plots.md:89]: Yes, we can. But probably we don't want to remove it, we want to add a paragraph about the detached experiments instead, WDYT?

Comment (tapadipti) [reply content/docs/studio/user-guide/projects-and-experiments/live-metrics-and-plots.md:89]: @mvshmakov yes, we should write about detached exps. It may also need a separate sub-section of its own. Can you pls share a rough draft of what to write (bullet points will be fine - I'll revise and include it).

Comment (mvshmakov) [reply content/docs/studio/user-guide/projects-and-experiments/live-metrics-and-plots.md:89]: * Detached experiments are the experiments, for those we cannot find a baseline commit in the remote repository. It can be: (1) local non-pushed commit; (2) commit that got removed by some mutative git action (e.g., rebase, hard reset with a push, squash commit, etc.).
* It is not a desired way to iterate on the experiments without having them assigned to a baseline commit.
* To "fix" them, one have to either push the missing commit to the git remote, or to remove the experiment from the UI and iterate on it once again with the correct baseline commit.
* To check which commit is a parent one, the user can do `git log --first-parent {sha1}`, where `sha1` is the copied exp SHA. I'm not sure if we should add it tbh, it must be a part of the UI - I'll send a PR.

File: content/docs/studio/user-guide/model-registry/add-a-model.md [hunk 2]
```diff
@@ -1,9 +1,8 @@
 # Add a model
```
Comment (tapadipti) [inline content/docs/studio/user-guide/model-registry/add-a-model.md:1]: The video included in this page (https://www.youtube.com/watch?v=szzv4ZXmYAs) must be updated, because `path to dvc.yaml` is missing in the video and the outdated `artifacts.yaml` is mentioned. @aguschin are there any other changes needed in the video?

Comment (tapadipti) [inline content/docs/studio/user-guide/model-registry/add-a-model.md:1]: @aguschin In point no. 4 below (`Enter the path of the model file as follows:`), 

        - First bullet: what if the model file is in remote but is tracked by dvc - which path to add here?
        - Second bullet: `repository path where the dvc reference to the model should be saved` How is this different from the path of `dvc.yaml`?

File: content/docs/studio/user-guide/model-registry/assign-stage.md [hunk 3]
```diff
@@ -67,7 +67,7 @@ assignment.
```
Comment (tapadipti) [inline content/docs/studio/user-guide/model-registry/assign-stage.md:67]: The video included in this page (https://youtu.be/Vrp1O5lkWBo) must be updated, because the `Use --force …` checkbox is missing in the video. @aguschin are there any other changes needed in the video?

File: content/docs/studio/user-guide/model-registry/use-models.md [hunk 4]
```diff
@@ -22,12 +24,13 @@ You can also construct this command manually using this template:
 $ dvc get ${GIT_REPO} ${MODEL_PATH} --rev ${GIT_REV}
 ```
 
-Let's download the model in the
```
Comment (tapadipti) [inline content/docs/studio/user-guide/model-registry/use-models.md:25]: @aguschin The `seg-model` model does not have an entry in `dvc.yaml` in the repo. And it does not show up in the project or MR if I add this repo in Studio. So, this example is confusing. Can we use an example that is already available in the demo model registry?

File: content/docs/studio/user-guide/team-collaboration/teams.md [hunk 5]
```diff
@@ -96,22 +93,38 @@ even if the project belongs to a team where you are an `Editor` or `Admin`.
 | Manage columns                               | No      | No     | Yes    | Yes   |
 | Manage custom files                          | No      | No     | Yes    | Yes   |
 
+### Privileges in the team's model registry
```
Comment (tapadipti) [inline content/docs/studio/user-guide/team-collaboration/teams.md:96]: @jellebouwman @aguschin Can you pls review this table of access permissions on MR for team members, and comment if anything is missing or incorrect. Pls also review the note at the end (`\*All team members can submit model actions...`).

File: content/docs/studio/user-guide/projects-and-experiments/live-metrics-and-plots.md (hunk 1 above)
Comment (dberenbaum) [reply content/docs/studio/user-guide/projects-and-experiments/live-metrics-and-plots.md:89]: > * To check which commit is a parent one, the user can do `git log --first-parent {sha1}`, where `sha1` is the copied exp SHA. I'm not sure if we should add it tbh, it must be a part of the UI - [I'll send a PR](https://github.com/iterative/studio/issues/6485).

I think it might be overkill to worry about what commit is the parent one, and it's not usual to push a specific commit. In most cases, suggesting that the user who generated it do a `git push` is probably enough.

File: content/docs/studio/user-guide/model-registry/remove-a-model-or-its-details.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+[indicate that the given model has been deregistered](https://mlem.ai/doc/gto/command-reference/deprecate#deregister-a-version).
+
+To **unassign a stage from a model version**, use the `Unassign stage` menu item
+in the 3-dot menu next to the model name in the models dashboard. You can also
+click on the the relevant stage assignment pill in the `Stages` section of the
+model details page to reveal the `Unassign stage` menu item. This action creates
+a new Git tag in your Git repository to
+[indicate that the given model version has been unassigned from the given stage](https://mlem.ai/doc/gto/command-reference/deprecate#unassigning-a-stage).
+
+Removing a Git tag from your Git repository also removes the model version or
```
Comment (tapadipti) [inline content/docs/studio/user-guide/model-registry/remove-a-model-or-its-details.md:25]: @aguschin Does it make sense to keep this paragraph or delete it? If we should keep it, it is complete and correct?

File: content/docs/studio/troubleshooting.md [hunk 7]
```diff
@@ -329,20 +326,20 @@ This error indicates that the `dvc.lock` file in the given commit has an invalid
...
 When there are updates (new commits, branches, etc.) in your Git repository,
 your project in Iterative Studio gets reflected to include those updates. If the
 project has stopped receiving updates from the Git repository and you have to
-"force import" the project each time to get any new commit, then it is possible
+`force import` the project each time to get any new commit, then it is possible
 that the Iterative Studio webhook in your repository got deleted or messed up.
 
 Iterative Studio periodically checks for any missing or messed up webhooks, and
-attempts to re-create it. Currently, this happens every 2 hours. The webhook
+attempts to re-create them. Currently, this happens every 2 hours. The webhook
```
Comment (tapadipti) [inline content/docs/studio/troubleshooting.md:342]: @amritghimire Are there any any steps that the user can manually take to resolve this? If yes, can you pls describe them?

File: content/docs/studio/troubleshooting.md [hunk 8]
```diff
@@ -308,15 +305,15 @@ Confirm that you are correctly following the
 [procedure to send live metrics and plots](/doc/studio/user-guide/projects-and-experiments/live-metrics-and-plots)
 to Iterative Studio.
 
-Also note that live metrics and plots for an experiment are displayed only if
-its parent Git commit is present in the project table. So, before you run the
-experiment, make sure that its parent commit is pushed to Git and shown in the
+Note that a live experiment is nested under the parent Git commit in the project
```
Comment (tapadipti) [inline content/docs/studio/troubleshooting.md:308]: @mvshmakov Is there anything else we need to explain here?

File: content/docs/studio/troubleshooting.md [hunk 9]
```diff
@@ -206,11 +203,11 @@ There are two possible reasons for this:
    columns that you select in the
    [**Columns** setting](/doc/studio/user-guide/projects-and-experiments/configure-a-project#columns).
 
-   **What if there are more than 200 columns?** Currently Iterative Studio
-   cannot import over 200 columns. If you have a large repository (with more
-   than 200 columns), one solution is to split the
+   **What if the repository has more than 200 columns?** Currently Iterative
```
Comment (tapadipti) [inline content/docs/studio/troubleshooting.md:206]: @amritghimire What is the current number for this limit (instead of 200)?

File: content/docs/studio/user-guide/model-registry/add-a-model.md (hunk 2 above)
Comment (aguschin) [reply content/docs/studio/user-guide/model-registry/add-a-model.md:1]: > In the first bullet point, what if the model file is in remote but is tracked by dvc - which path to add here?

Path to the tracked file in the Git repo. Regardless of file location in remote, if the model was saved to `some/path` (inside the repo), you should enter `some/path`.

> In the second bullet point, repository path where the dvc reference to the model should be saved How is this different from the path of dvc.yaml?

Let's assume I have my model in `s3://bucket/file`. Studio BE will run `dvc import` (or `dvc import-url`, I'm not sure), creating a `.dvc` file inside the repo. The path for this file is what you're asking about. You can see it's shown in "add a model" once you specify remote path.

Jun-27-2023 15-27-54

File: content/docs/studio/user-guide/model-registry/assign-stage.md (hunk 3 above)
Comment (aguschin) [reply content/docs/studio/user-guide/model-registry/assign-stage.md:67]: models table and MDP have changed a lot, so it will be easier to re-record it completely I guess.

File: content/docs/studio/user-guide/model-registry/use-models.md (hunk 4 above)
Comment (aguschin) [reply content/docs/studio/user-guide/model-registry/use-models.md:25]: Right. I need to fix that. @daavoo, did you complete updating example-get-started already? I guess I need to add some GTO command calls in generate.sh.

File: content/docs/studio/user-guide/model-registry/remove-a-model-or-its-details.md (hunk 6 above)
Comment (aguschin) [reply content/docs/studio/user-guide/model-registry/remove-a-model-or-its-details.md:25]: @tapadipti, I'm not seeing this page in docs at all. Wasn't it deleted from the sidebar?

File: content/docs/studio/user-guide/model-registry/use-models.md (hunk 4 above)
Comment (daavoo) [reply content/docs/studio/user-guide/model-registry/use-models.md:25]: `example-get-started-experiments` , yes, please add to `generate.sh` so we can keep updating on top without worrying

File: content/docs/studio/troubleshooting.md (hunk 8 above)
Comment (mvshmakov) [reply content/docs/studio/troubleshooting.md:308]: As per the https://github.com/iterative/dvc.org/pull/4638#discussion_r1242248953, should we explicitly say that the user should try `git push`?

Apart from it, looks good to me!

File: content/docs/studio/user-guide/projects-and-experiments/live-metrics-and-plots.md (hunk 1 above)
Comment (tapadipti) [reply content/docs/studio/user-guide/projects-and-experiments/live-metrics-and-plots.md]: Thanks @mvshmakov @dberenbaum.
I've added a section for detached experiments [here](https://github.com/iterative/dvc.org/pull/4638/commits/ede1f93a174f68d79eddfc4e3e6c9171940d1b76).

File: content/docs/studio/user-guide/model-registry/remove-a-model-or-its-details.md (hunk 6 above)
Comment (tapadipti) [reply content/docs/studio/user-guide/model-registry/remove-a-model-or-its-details.md:25]: @aguschin This is a new page created in this PR.

File: content/docs/studio/user-guide/model-registry/use-models.md (hunk 4 above)
Comment (aguschin) [reply content/docs/studio/user-guide/model-registry/use-models.md:25]: @tapadipti - I made a commit to address these changes.

File: content/docs/studio/user-guide/model-registry/add-a-model.md (hunk 2 above)
Comment (tapadipti) [reply content/docs/studio/user-guide/model-registry/add-a-model.md:1]: Thanks @aguschin.
I've made a small change [here](https://github.com/iterative/dvc.org/pull/4638/commits/a00cd3e8ff07dc0b50bdeb0494ce6bba858251ea). Let me know if it doesn't make sense.

File: content/docs/studio/troubleshooting.md (hunk 8 above)
Comment (tapadipti) [reply content/docs/studio/troubleshooting.md:308]: Added [here](https://github.com/iterative/dvc.org/pull/4638/commits/19e4cc2cadef107080637f1ae384ee590d7e49c5)

File: content/docs/studio/user-guide/model-registry/remove-a-model-or-its-details.md [hunk 10]
```diff
@@ -0,0 +1,30 @@
...
+stage assignment represented by that Git tag. To get this change to reflect in
+Iterative Studio, reparse the repository by opening the project in Iterative
+Studio and clicking on `Force import` as shown below.
+
+![Showing and hiding columns](https://static.iterative.ai/img/studio/force_import.gif)
```
Comment (aguschin) [inline content/docs/studio/user-guide/model-registry/remove-a-model-or-its-details.md:30]:
```suggestion
stage assignment represented by that Git tag.
```

File: content/docs/studio/user-guide/model-registry/remove-a-model-or-its-details.md (hunk 6 above)
Comment (aguschin) [reply content/docs/studio/user-guide/model-registry/remove-a-model-or-its-details.md:25]: Sorry, didn't see it first. You don't need to do force import. GH will call webhook and Studio BE will update this.

File: content/docs/studio/user-guide/model-registry/add-a-model.md (hunk 2 above)
Comment (aguschin) [reply content/docs/studio/user-guide/model-registry/add-a-model.md:1]: LGTM!

File: content/docs/studio/troubleshooting.md (hunk 9 above)
Comment (amritghimire) [reply content/docs/studio/troubleshooting.md:206]: It is by default 5k. But, probably changed by @iterative/platform

Comment (amritghimire) [reply content/docs/studio/troubleshooting.md:206]: It relies on PARSING_LIMITS

File: content/docs/studio/troubleshooting.md (hunk 7 above)
Comment (amritghimire) [reply content/docs/studio/troubleshooting.md:343]: Force import should take care of this.

Comment (tapadipti) [reply content/docs/studio/troubleshooting.md:343]: @amritghimire Does this mean that if the webhook is messed up and the user has to run force import to update the project table, then from the next time onwards the project table will get updated automatically? (coz the webhook got recreated when the user ran force import for the first time)?

File: content/docs/studio/troubleshooting.md (hunk 9 above)
Comment (jesper7) [reply content/docs/studio/troubleshooting.md:206]: We use default `PARSING_LIMITS` value in production, so 5k

Comment (tapadipti) [reply content/docs/studio/troubleshooting.md:206]: Thanks @amritghimire @jesper7

**Comment by tapadipti** [comment]
@iterative/docs Can I get a review for this PR pls. I'd like to merge it soon.

File: content/docs/studio/get-started.md [hunk 11]
```diff
@@ -59,9 +59,8 @@ details]).
 
    ![](https://static.iterative.ai/img/studio/view_components.png)
 
-6. You can [train on the cloud and submit new experiments] by changing
-   hyperparameters and datasets, and you can run on your own cloud
-   infrastructure.
+6. You can [submit new experiments] by changing hyperparameters and datasets,
+   and you can run the experiments on your own cloud infrastructure.
```
Comment (dberenbaum) [inline content/docs/studio/get-started.md:63]: Why change this? I think I added this pretty recently to stress that it's not only about changing hyperparameters and running experiments. You could use this workflow for any training on the cloud.

**Comment by tapadipti** [comment]
> @tapadipti Can you provide a high-level summary of the changes and motivation behind them?

@dberenbaum As mentioned in the description above, this PR partially addresses https://github.com/iterative/studio/issues/2244 (except the screenshots and some other points). Pls take a look at the checked items from the list in that issue.

This issue has been open for a long time, and I'm trying to close it now.

File: content/docs/studio/get-started.md (hunk 11 above)
Comment (tapadipti) [reply content/docs/studio/get-started.md:63]: This one is just a rewording. I couldn't understand `and you can run on your own cloud infrastructure`. It probably means `and you can run (the training?) on your own cloud infrastructure`. Does this sentence convey your message `You can train on the cloud, including on your own cloud infrastructure, and submit new experiments by changing hyperparameters and datasets.`?

**Comment by tapadipti** [comment]
@dberenbaum @iterative/docs Let me know if you need me to split this PR into multiple PRs, one for each kind of change. If that will help review this faster, I'll be happy to spend some time on it, so that this can be closed soon.

File: content/docs/studio/troubleshooting.md (hunk 9 above)
Comment (tapadipti) [reply content/docs/studio/troubleshooting.md]: @amritghimire @jesper7 I can see that the PARSING_LIMITS was changed to 2k in the itops repo, and that change was later reverted. In the Studio repo, the PARSING_LIMITS has different values for soft/hard/all_columns. Can you pls point to which configuration gets finally applied?

**Comment by dberenbaum** [comment]
@tapadipti I will take a look and let you know. 

I still am not sure whether the items in https://github.com/iterative/studio/issues/2244 are p1. Is it just about closing old issues? I'm fine to review this time since it's already done, but I don't know that we should continue to prioritize similar issues.

File: content/docs/studio/get-started.md (hunk 11 above)
Comment (dberenbaum) [reply content/docs/studio/get-started.md:63]: Yes, sounds good, @tapadipti

File: content/docs/studio/troubleshooting.md [hunk 12]
```diff
@@ -78,24 +75,24 @@ import the repo again.
...
 Azure, Google Drive, Google Cloud Storage and SSH.
 
-## Errors related to credentials
-
-If your repository uses data remotes that Iterative Studio needs to access, then
-you should [add the required credentials to your project][cloud-credentials]. If
-credentials are missing or incorrect, you will see one of the following errors:
+If the data remotes have access control, then you should [add the required
+credentials to your project][cloud-credentials]. If credentials are missing or
+incorrect, you will see one of the following errors:
```
Comment (dberenbaum) [inline content/docs/studio/troubleshooting.md:95]: Why the changes here? The old wording makes more sense to me.

File: content/docs/studio/troubleshooting.md [hunk 13]
```diff
@@ -359,29 +358,11 @@ For a model to be displayed in the model registry, it has to be registered using
...
-that you want to un-assign.
-
-After removing the Git tag, reparse the repository. To do this, open the project
-in Iterative Studio and then click on `Force import` as shown below.
-
-![Showing and hiding columns](https://static.iterative.ai/img/studio/force_import.gif)
-
-Currently, you cannot un-assign stages from the Iterative Studio user interface.
+When a project is deleted from the projects dashboard, all its models get
+automatically removed form the model registry. So make sure that the project is
```
Comment (dberenbaum) [inline content/docs/studio/troubleshooting.md:364]:
```suggestion
automatically removed from the model registry. So make sure that the project is
```

File: content/docs/studio/troubleshooting.md [hunk 14]
```diff
@@ -359,29 +358,11 @@ For a model to be displayed in the model registry, it has to be registered using
...
+not deleted.
```
Comment (dberenbaum) [inline content/docs/studio/troubleshooting.md:365]: It sounds more dangerous to me than it really is. If they re-add the project, everything will reappear without losing any info, right? Should we clarify this and make clear that the git tags won't disappear, so everything is safe as long as the git remote is intact?

File: content/docs/studio/user-guide/account-management.md [hunk 15]
```diff
@@ -46,51 +47,36 @@ next to the email address which you want to designate as primary.
...
-  Studio GitHub app on additional organizations or repositories, or even remove
-  the app from organizations or repositories where you no longer need it.
-
-## Cloud credentials
-
-Here, you can see all the credentials you have added for your cloud and data
-remotes. You can also add new credentials as well as edit or delete existing
-credentials. These credentials useful in two cases:
+In the `Cloud Credentials` section of your [Profile] page, you can view, add and
+update credentials for cloud resources. These credentials are used to
```
Comment (dberenbaum) [inline content/docs/studio/user-guide/account-management.md:70]:
```suggestion
update credentials for cloud resources. These credentials are used to:
```

File: content/docs/studio/user-guide/model-registry/remove-a-model-or-its-details.md (hunk 6 above)
Comment (dberenbaum) [reply content/docs/studio/user-guide/model-registry/remove-a-model-or-its-details.md:25]: Are screen shots a follow up item here? Feels like they would be way more useful than text.

File: content/docs/studio/user-guide/model-registry/use-models.md (hunk 4 above)
Comment (dberenbaum) [reply content/docs/studio/user-guide/model-registry/use-models.md:25]: @aguschin There is now a `pool-segmentation` model available, right? Why change this to use the other repo?

File: content/docs/studio/user-guide/projects-and-experiments/configure-a-project.md [hunk 16]
```diff
@@ -8,61 +8,45 @@ may be required depending on how your Git repository has been set up.
...
 
 ### Non-DVC repositories
 
-In the section on [preparing your repositories], you saw that you can use
-Iterative Studio with DVC as well as non-DVC repositories. If you are connecting
-to a non-DVC repository, then you will need to [specify the custom files] that
-contain the metrics and hyperparameters that you want to visualize.
+If you are creating a project for a non-DVC repository, you will need to
+[specify the custom files] that contain the metrics and hyperparameters that you
+want to visualize.
```
Comment (dberenbaum) [inline content/docs/studio/user-guide/projects-and-experiments/configure-a-project.md:18]: I thought we were dropping this scenario?

File: content/docs/studio/user-guide/projects-and-experiments/explore-ml-experiments.md [hunk 17]
```diff
@@ -97,50 +89,50 @@ from the table.
...
-  `[skip studio]` in the commit message of the first commit.
-
-- **Hide commits and branches manually:** You can selectively hide commits and
-  branches. This can be useful if there are commits that do not add much value
-  in your project. To hide a commit or branch, click on the 3-dot menu next to
-  the commit or branch name and click on `Hide commit` or `Hide branch`.
+  with the experiment results (metrics and plots). You may want to hide the
+  first commit and only display the second commit, which has the new values for
+  the hyper-parameters as well as experiment results. For this, you can use the
+  string `[skip studio]` in the commit message of the first commit.
```
Comment (dberenbaum) [inline content/docs/studio/user-guide/projects-and-experiments/explore-ml-experiments.md:95]: Very minor, but the example scenario here is so specific to the way our CI workflow works that I doubt users understand. It might be simpler to use an example like "suppose you create a Git commit with only code changes unrelated to your model."

File: content/docs/studio/user-guide/projects-and-experiments/run-experiments.md [hunk 18]
```diff
@@ -106,81 +106,78 @@ set up your CI.
...
+
+     | Parameter    | Meaning                                                                                                                                                                                                                                                                                                                                                               |
+     | ------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+     | `Job script` | This is the script needed for your runner to execute your job, which would commonly include training your model. The default template is a very common combination of CML and DVC taking into account that DVC enables you to make the most of Iterative Studio. You can update this script to reflect your exact model training process, whether you use DVC or not. |
```
Comment (dberenbaum) [inline content/docs/studio/user-guide/projects-and-experiments/run-experiments.md:136]: Related to https://github.com/iterative/studio/issues/6512, let's not spend too much time here, but maybe we should mention the need for a token and to configure your cloud credentials to launch the runner and access storage.

**Review by dberenbaum — CHANGES_REQUESTED** [review]
Thanks @tapadipti. I have a few questions and comments but nothing major.

File: content/docs/studio/user-guide/model-registry/remove-a-model-or-its-details.md (hunk 6 above)
Comment (tapadipti) [reply content/docs/studio/user-guide/model-registry/remove-a-model-or-its-details.md:25]: Yes, I'll raise a new PR with these screenshots as well as updates to other outdated ones.

File: content/docs/studio/troubleshooting.md (hunk 9 above)
Comment (tapadipti) [reply content/docs/studio/troubleshooting.md]: @mvshmakov Since you are working on https://github.com/iterative/studio/issues/6058, can you pls confirm the numbers here?

File: content/docs/studio/troubleshooting.md (hunk 12 above)
Comment (tapadipti) [reply content/docs/studio/troubleshooting.md:95]: To manage the total number of items at the top on the troubleshooting page, I merged the sections `Errors related to DVC remotes` and `Errors related to credentials` into a single section `Errors related to DVC remotes and credentials`. I did the rewording to make the content within this single section flow better from one paragraph to next (the previous paragraph already mentions `Iterative Studio can include data from data remotes..`).

File: content/docs/studio/get-started.md (hunk 11 above)
Comment (tapadipti) [reply content/docs/studio/get-started.md]: Done [here](https://github.com/iterative/dvc.org/pull/4638/commits/8116b516fad73912cf266d3121e19f46b9c63846)

File: content/docs/studio/user-guide/projects-and-experiments/configure-a-project.md (hunk 16 above)
Comment (tapadipti) [reply content/docs/studio/user-guide/projects-and-experiments/configure-a-project.md:18]: So far, we still have it.

File: content/docs/studio/user-guide/projects-and-experiments/explore-ml-experiments.md (hunk 17 above)
Comment (tapadipti) [reply content/docs/studio/user-guide/projects-and-experiments/explore-ml-experiments.md:95]: Good point. But the earlier bullet point (`Iterative Studio auto-hides irrelevant commits`) already covers the scenario that you have mentioned as it says `Iterative Studio identifies commits where metrics, files and hyperparameters did not change and hides them automatically.` So, that example would not be enough.

Because `[skip studio]` was introduced specifically for the scenario where there's one commit with just hyperparam changes and another commit which also includes the results, I think it's ok to use the current example (although I agree it's not very easy to understand).

File: content/docs/studio/user-guide/projects-and-experiments/run-experiments.md (hunk 18 above)
Comment (tapadipti) [reply content/docs/studio/user-guide/projects-and-experiments/run-experiments.md:136]: Sure. I'll do it as part of https://github.com/iterative/studio/issues/6512.

File: content/docs/studio/troubleshooting.md (hunk 14 above)
Comment (tapadipti) [reply content/docs/studio/troubleshooting.md]: You're right. I've updated it [here](https://github.com/iterative/dvc.org/pull/4638/commits/e2e893cbe00d960669980e0c9cd8a835628f3ea4).

**Comment by tapadipti** [comment]
Thanks for the review @dberenbaum. I've addressed your comments.

File: content/docs/studio/troubleshooting.md (hunk 9 above)
Comment (mvshmakov) [reply content/docs/studio/troubleshooting.md]: The default parsing limits in `docker-compose/base.yaml` is `{"commits":{"soft":3000,"hard":5000},"columns":{"soft":100,"hard":200},"all_columns":3000}`.

The default value that backend fallbacks to if nothing was specified at all is `{"commits": {"soft": 3000, "hard": 5000},"columns": {"soft": 200, "hard": 200},"all_columns": 5000}`.

> In the Studio repo, the PARSING_LIMITS has different values for soft/hard/all_columns. Can you pls point to which configuration gets finally applied?

From the docs:

```txt
Soft limits: max amount of commits / columns to import during the first parse.
Hard limits: max amount of commits / columns to import on subsequent parses.
All columns: max amount of entries in files/params/metrics to track for display
```

So, depending on whether we apply `base.yaml` env variables, it is either 3000 or 5000. @iterative/platform do we propagate `docker-compose/base.yaml` env variables by default when deploying to the production?

Comment (mjasion) [reply content/docs/studio/troubleshooting.md]: No, `docker-compose` configuration is separate from prod. To propagate someone has to do this in `config.env` files 
- [prod](https://github.com/iterative/itops/blob/main/k8s/prod/studio/config.env)
- [prod-uk](https://github.com/iterative/itops/blob/main/k8s/uk/studio/default/config.env)
- [dev](https://github.com/iterative/itops/blob/main/k8s/dev/studio/config.env)

File: content/docs/studio/user-guide/projects-and-experiments/configure-a-project.md (hunk 16 above)
Comment (dberenbaum) [reply content/docs/studio/user-guide/projects-and-experiments/configure-a-project.md:18]: @tapadipti I think we have already committed to dropping it in https://github.com/iterative/studio/issues/5957 and have started taking action on it in https://github.com/iterative/studio/pull/6293. I don't see much downside to dropping it here.

**Review by dberenbaum — APPROVED** [review]
Thanks @tapadipti! I left one comment (I still think we should drop custom metrics) but it's not a blocker, so feel free to merge when ready.

**Comment by tapadipti** [comment]
> I left one comment (I still think we should drop custom metrics) but it's not a blocker, so feel free to merge when ready.

Thanks @dberenbaum. I've created a [new issue](https://github.com/iterative/dvc.org/issues/4678) for this so as not to prolong the current PR further.
